bookshelf/src/main/webapp/base.jsp

```diff
@@ -34,9 +34,11 @@ Copyright 2015 Google Inc. All Rights Reserved.
         </div>
         <ul class="nav navbar-nav">
           <li><a href="/">Books</a></li>
-          <li><a href="/books/mine">My Books</a></li>
+          <div class="form-group ${isAuthConfigured ? '' : 'hidden'}">
```

Comment:
Oh, but we aren't guaranteed that ListServlet will be the first one called. I think I will make a helper utility and make sure it's called from each.